okcurl/build.gradle

```
@@ -1,3 +1,9 @@
+plugins {
+  id "com.palantir.graal" version "0.7.1-20-g113a84d"
```

Just a heads up this is technically a snapshot build that was accidentally publishing (see https://github.com/palantir/gradle-graal/pull/392 ). The [only changes since 0.7.1 release are updates to build plugins](https://github.com/palantir/gradle-graal/compare/0.7.1...develop), so I think you could use 0.7.1 here if you like until a 0.7.2 or 0.8.0 are cut.


```suggestion
  id "com.palantir.graal" version "0.7.1"
```

---

Fixed.